internal/server/providers/claudeprojects/claudeprojects_e2e_test.go

```
@@ -0,0 +1,393 @@
+	return resp.Data.Conversation
+}
+
+// postQuery is the GraphQL transport for tests. Identical shape to the
+// CLI's runRaw but without pretty-printing.
+func postQuery(t *testing.T, baseURL, query string) graphqlResponse {
+	t.Helper()
+	body, err := json.Marshal(map[string]string{"query": query})
+	if err != nil {
+		t.Fatalf("marshal request: %v", err)
+	}
+	req, err := http.NewRequest(http.MethodPost, baseURL+"/graphql", strings.NewReader(string(body)))
```

golangci-lint (2.12.1)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)